MTM-63992 Support URL-encoding any character in returned (self)links (#3565)
SoftwareAG/c8y-docs: content/change-logs/platform-services/cumulocity-2025-247-0-strict-url-encoding-in-self-links.md

---
date: ''

> title: Strict URL encoding in self-links

Reviewer: ```suggestion
title: Improved URL encoding in self-links
```

---
date: ''
title: Strict URL encoding in self-links
product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-OG_650_b2
    label: Core platform
build_artifact:
  - value: tc-QHwMfWtBk7
    label: cumulocity
ticket: MTM-63992
version: 2025.247.0
---

> Encoding of special characters in self-links has been improved to ensure that they are URL-encoded correctly.

Reviewer: ```suggestion
```

> This change addresses an issue where self-links containing special characters were not properly encoded, leading to broken links.

Reviewer: ```suggestion
Self-links containing special characters were not properly encoded, leading to broken links. This issue has been fixed. The encoding of special characters in self-links has been improved to ensure that the self-links are URL-encoded correctly.
```
